connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -904,6 +905,29 @@ message ApplyInPandasWithState {
   string timeout_conf = 7;
 }
 
+message FlatMapGroupsWithState {
+  // (Required) One input relation for CoGroup Map API - applyInPandas.
```

Comment:
the comment seems not correct

Reply:
Sorry forgot to update this, will update in next revision since I'll also need to fix some merge conflicts